src/config.py

```diff
@@ -31,3 +40,69 @@ def is_benchmark_mode_enabled():
 
 def is_build_enabled():
     return parse_dlc_developer_configs("build", "do_build")
+
+
+def is_ec2_test_enabled():
+    return parse_dlc_developer_configs("test", "ec2_tests")
+
+
+def is_ecs_test_enabled():
+    return parse_dlc_developer_configs("test", "ecs_tests")
+
+
+def is_eks_test_enabled():
+    return parse_dlc_developer_configs("test", "eks_tests")
+
+
+def is_sanity_test_enabled():
+    return parse_dlc_developer_configs("test", "sanity_tests")
+
+
+def is_sm_local_test_enabled():
+    return parse_dlc_developer_configs("test", "sagemaker_local_tests")
+
+
+def are_efa_tests_enabled():
+    return parse_dlc_developer_configs("test", "efa_tests")
+
+
+def is_scheduler_enabled():
+    return parse_dlc_developer_configs("test", "use_scheduler")
+
+
+class AllowedSMRemoteConfigValues(Enum):
+    OFF = "off"
+    RC = "rc"
+    DEFAULT = "default"
+
+
+def get_sagemaker_remote_tests_config_value():
+    """
+    Get the actual config option for sm remote tests
+    """
+    return parse_dlc_developer_configs("test", "sagemaker_remote_tests")
+
+
+def is_sm_remote_test_enabled():
+    """
+    Check to see if sm remote test is enabled by config
+    """
+    sm_remote_tests_value = get_sagemaker_remote_tests_config_value()
+    allowed_values = [cfg_opt.value for cfg_opt in AllowedSMRemoteConfigValues]
+
+    # Sanitize value in case of extra whitespace or inconsistent capitalization
+    if isinstance(sm_remote_tests_value, str):
+        sm_remote_tests_value = sm_remote_tests_value.lower().strip()
+
+    if sm_remote_tests_value == AllowedSMRemoteConfigValues.OFF.value:
```

Comment:
We don't even need that check. You return False by default.

Reply:
We should add this check so that we don't warn when this is set to a valid "off" value

Reply:
How about this - I'll get rid of the condition, but add a condition to "warn" if it's not set to OFF

Reply:
Yes. You right. 
Sure that works also